src/CMakeLists.txt

```diff
@@ -544,6 +517,7 @@ include_directories(
 	${JSON_INCLUDE_DIR}
 	${X11_INCLUDE_DIR}
 	${PROJECT_SOURCE_DIR}/script
+	${PROJECT_SOURCE_DIR}/client
```

Comment:
don't include client folder, include it explicitely permit to track which code should be moved to client folder and separated from common files